kubernetes/test-infra/gke/main.tf

```diff
@@ -18,11 +18,14 @@ resource "random_id" "password" {
 
 # See https://cloud.google.com/container-engine/supported-versions
 variable "kubernetes_version" {}
+variable "workers_count" {
+  default = "4"
```

Comment:
Is it really desired to change the default from 3 to 4?

With n1-standard-4 instances, this will go over the default GCP 24 CPU core per region quota.

Reply:
It’s not a requirement, just aligning it to what the others are set to.
I wasn’t aware of the quota thing, since we’re running the tests in paid accounts and I didn’t run into it.
I can drop it back to 3 and override it in CI.

Reply:
I too run tests in paid accounts, but 24 is the default and going higher requires a request to Google support.